extensions/sources/consul/src/main/java/io/smallrye/config/source/consul/ConsulConfigSource.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.smallrye.config.source.consul;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.logging.Level;
+import java.util.stream.Collectors;
+
+import org.apache.commons.text.StringSubstitutor;
+
+import com.ecwid.consul.v1.ConsulClient;
+import com.ecwid.consul.v1.kv.model.GetValue;
+import io.smallrye.config.source.EnabledConfigSource;
+
+import lombok.extern.java.Log;

```

Comment:
Are we going to start relying on Lombok?

Reply:
In another comment I said we'd likely remove that, either in the PR or shortly after

Reply:
Lombok is removed.